pymess/migrations/0028_migration.py

```diff
@@ -14,4 +14,9 @@ class Migration(migrations.Migration):
             field=models.BooleanField(default=False),
             preserve_default=False,
         ),
+        migrations.AlterField(
+            model_name='smstemplate',
+            name='is_secret',
+            field=models.BooleanField(default=True),
```

Comment:
second operation sets on the model default value = True. No db changes.
It has to be modification of this existing migration file otherwise if django chooses a different migration path it could fail.
E.g.
1. pymess 28
2. from app changing SMSTemplate object (fail requires is_secret field)
3. pymess 29 making field default